tasks/notify.py

```diff
@@ -171,3 +177,66 @@ def print_gitlab_ci(
 
     # Print
     print_gitlab_ci_configuration(yml, sort_jobs=sort)
+
+
+@task
+def gitlab_ci_diff(ctx, before: str | None = None, after: str | None = None, pr_comment: bool = False):
+    """
+    Creates a diff from two gitlab-ci configurations.
+
+    - before: Git ref without new changes, None for LCA commit between HEAD and main
+    - after: Git ref with new changes, None for current local configuration
+    - pr_comment: If True, post the diff as a comment in the PR
+    - NOTE: This requires a full api token access level to the repository
+    """
+
+    pr_comment_head = 'Gitlab CI Configuration Changes'
+    job_url = os.environ['CI_JOB_URL']
```

Comment:
This makes the task hard to use locally, you have to set the `CI_JOB_URL` locally, even though it's only used in the case where a PR comment is made.

Reply:
Fixed 👍